modules/storage-persistent-storage-pv.adoc

```diff
@@ -301,3 +301,31 @@ ifndef::openshift-dedicated,openshift-rosa[]
 Fibre Channel and HostPath PVs do not support mount options.
 ====
 endif::openshift-dedicated,openshift-rosa[]
+endif::microshift[]
+
+ifdef::microshift[]
+.Mount options example
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: topolvm-provisioner
+mountOptions:
+  - uid=1500
+  - gid=1500
+parameters:
+  csi.storage.k8s.io/fstype: xfs
+provisioner: topolvm.io
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+----
+
+[NOTE]
+====
+`mountOptions`are not validated. Incorrect values will cause the mount to fail and an event to be logged to the PVC.
```

Comment:
I think `mountOptions` is not rendering correctly in the preview because there's a missing space between that and "are"

```suggestion
`mountOptions` are not validated. Incorrect values will cause the mount to fail and an event to be logged to the PVC.
```